### DISCUSSION TITLE
Enable Github Discussions for memcode

### DISCUSSION THREAD

**Discussion by kgashok** [lakesare/memcode#140] [Ideas]
I was reading this article about Github discussions - https://github.blog/2022-01-13-how-five-open-source-communities-are-using-github-discussions/. 

I presume this can be a complement to the memcode slack channel.

**Comment by lakesare** [comment]
Very interesting! I did want something like this, where people can upvote the features.

I noticed it's already enabled, even though I didn't do it.

<img width="321" alt="image" src="https://user-images.githubusercontent.com/7578559/149614545-946f66ce-e287-448f-9d84-8ad8bc1f5afa.png">
